src/screens/send/components/gas-fee-advanced/gas-fee-advanced.tsx

```diff
@@ -158,7 +159,7 @@ export class GasFeeAvancedComponent extends React.Component<
                     >
                         {translate('Fee.gasLimit')}
                     </Text>
-                    <Text style={styles.gasPriceUnit}>{`(${gasPriceUnit})`}</Text>
+                    <Text style={styles.gasPriceUnit}>{`(${defaultUnit})`}</Text>
```

Comment:
do we really want this ?

Reply:
@krisboit Gas price should be in GWEI and Gas limit in WEI correct? - both ar displayed correctly but the label above the field says GWEI on both